src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

```diff
@@ -41,6 +41,7 @@
                 </div>
                 <div class="d-flex align-items-center">
                     @await RenderSectionAsync("NavbarTop", required: false)
+                    @await DisplayAsync(await New.CulturePicker(), "OrchardCore.Localization.CulturePicker.Admin")
```

Comment:
I just added another overload to make the display happen when a feature is on. @Piedone the culture picker should works now without any issue

I might split the `DisplayAsync()` changes into a separated PR

Reply:
#12030

Reply:
I don't like it, TheAdmin has a dependency on the other feature. The feature should inject the shape.